### PR TITLE
CMOS-224: Document CMOS' own health checks

### PR DESCRIPTION
**Pull request by markspolakovs**
Merge together the docs from cbmultimanager and observability into a single file that contains documentation from both. Update the build process accordingly, including special handling for OSS and GitHub Pages builds.

### CODE REVIEW

**Comment by markspolakovs** [comment]
@maria-robobug I suspect you'll need to add cbmultimanager to the docs-site config _before_ this PR is merged, otherwise it'll refer to the cbmultimanager partials which aren't available yet - let me know once that's in place and I'll merge this PR. Thanks!

**Comment by maria-robobug** [comment]
Will update you soon @markspolakovs, just having some issues building staging at the moment so trying to fix. I'll comment here once resolved.

File: microlith/grafana/provisioning/dashboards/bucket-overview.json [hunk 1]
```diff
@@ -406,7 +406,397 @@
           "custom": {
             "align": "center",
             "displayMode": "auto",
-            "minWidth": 1
+            "minWidth": 1,
```
Comment (redm2) [inline microlith/grafana/provisioning/dashboards/bucket-overview.json:409]: Where's this change coming from? The rest are just URL changes, is this a diff failure?

Comment (markspolakovs) [reply microlith/grafana/provisioning/dashboards/bucket-overview.json:409]: Merge conflict, if I recall correctly - have I merged it the wrong way round?

Comment (redm2) [reply microlith/grafana/provisioning/dashboards/bucket-overview.json:409]: Hmm no should be okay!

**Review by redm2 — APPROVED** [review]

**Comment by maria-robobug** [comment]
@markspolakovs done, staging is currently building. Feel free to merge this.

**Comment by markspolakovs** [comment]
@maria-robobug perfect, thanks ever so much for your help!

**Pull request merged**